tests/test_crm.py

```diff
@@ -720,6 +720,60 @@ def test_CrmDnatEntry(self, dvs, testlog):
         assert used_counter == 0
         assert avail_counter != 0
 
+    def test_CrmResetThresholdExceedCount(self, dvs, testlog):
+
+        config_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+        intf_tbl = swsscommon.Table(config_db, "INTERFACE")
+        fvs = swsscommon.FieldValuePairs([("NULL","NULL")])
+        intf_tbl.set("Ethernet0", fvs)
+        intf_tbl.set("Ethernet0|10.0.0.0/31", fvs)
+        dvs.runcmd("config interface startup Ethernet0")
+
+        dvs.runcmd("crm config polling interval 1")
```

Comment:
@link19430 , please use `crm_update(dvs, "polling_interval", "1")` and fix all the CLI command usage. VS tests are modified *not* use CLI commands. You can take a look at other tests for reference.

Reply:
all cli commands had modified to use crm_update()